package.json

```diff
@@ -65,7 +64,8 @@
     "yup": "1.0.0-beta.3"
   },
   "resolutions": {
-    "@openzeppelin/**/@openzeppelin/upgrades-core": "^1.14.1"
+    "@openzeppelin/**/@openzeppelin/upgrades-core": "^1.14.1",
+    "**/@ethersproject/contracts": "5.6.0"
```

Comment:
was this needed after amie bumped these in the monorepo?

Reply:
Yes, this was done to make her change work.  Yarn was hoisting the older version still without this.